src/iOS/toga_iOS/libs/core_graphics.py

```diff
@@ -1,6 +1,114 @@
 ##########################################################################
 # System/Library/Frameworks/CoreGraphics.framework
 ##########################################################################
+from ctypes import *
+from ctypes import util
+
+from rubicon.objc.types import register_preferred_encoding
+from rubicon.objc import *
+
+
+######################################################################
+core_graphics = cdll.LoadLibrary(util.find_library('CoreGraphics'))
+######################################################################
+
+######################################################################
+# CGAffineTransform.h
+
+
+class CGAffineTransform(Structure):
+    _fields_ = [
+        ("a", CGFloat),
+        ("b", CGFloat),
+        ("c", CGFloat),
+        ("d", CGFloat),
+        ("tx", CGFloat),
+        ("ty", CGFloat),
+    ]
+
+
+core_graphics.CGAffineTransformIdentity = CGAffineTransform
+core_graphics.CGAffineTransformInvert.restype = CGAffineTransform
+core_graphics.CGAffineTransformInvert.argtypes = [CGAffineTransform]
+core_graphics.CGAffineTransformMakeScale.restype = CGAffineTransform
+core_graphics.CGAffineTransformMakeScale.argtypes = [CGFloat, CGFloat]
+
+######################################################################
+# CGContext.h
+CGContextRef = c_void_p
+register_preferred_encoding(b'^{__CGContext=}', CGContextRef)
+
+CGPathDrawingMode = c_int32
+kCGPathFill = 0
+kCGPathEOFill = 1
+kCGPathStroke = 2
+kCGPathFillStroke = 3
+kCGPathEOFillStroke = 4
+
+CGTextDrawingMode = c_int32
+kCGTextFill = 0
+kCGTextStroke = 1
+kCGTextFillStroke = 2
+kCGTextInvisible = 3
+kCGTextFillClip = 4
+kCGTextStrokeClip = 5
+kCGTextFillStrokeClip = 6
+kCGTextClip = 7
+
+CGTextEncoding = c_int32
+kCGEncodingFontSpecific = 0
+kCGEncodingMacRoman = 1
+
+core_graphics.CGContextAddArc.restype = c_void_p
+core_graphics.CGContextAddArc.argtypes = [CGContextRef, CGFloat, CGFloat, CGFloat, CGFloat, CGFloat, c_int]
```

Comment:
At column 57: (F405) 'CGFloat' may be undefined, or defined from star imports: ctypes, rubicon.objc